pymem/__init__.py

```diff
@@ -308,7 +308,7 @@ def free(self, address):
             raise pymem.exception.ProcessError('You must open a process before calling this method')
         return pymem.memory.free_memory(self.process_handle, address)
 
-    def pattern_scan_all(self, pattern, *, return_multiple=False):
+    def pattern_scan_all(self, pattern, *, return_multiple=False, scan_hex=False):
```

Comment:
Flag added to the existing scan method rather than creating a new method for the nearly same purpose